frontend/src/metabase/components/Editor/highlight.ts

```diff
@@ -0,0 +1,36 @@
+import { HighlightStyle } from "@codemirror/language";
```

Comment:
This might not be the right location for this file.
I'm planning to move more shared codemirror editor stuff to this folder in follow up PR's.